runelite-client/src/main/java/net/runelite/client/plugins/nightmarezone/NightmareZoneConfig.java

```diff
@@ -47,11 +47,55 @@ default boolean moveOverlay()
 		return true;
 	}
 
+	@ConfigItem(
+		keyName = "powersurgenotification",
+		name = "Power surge notification",
+		description = "Toggles notifications when a power surge power-up appears",
+		position = 2
+	)
+	default boolean powerSurgeNotification()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "recurrentdamagenotification",
+		name = "Recurrent damage notification",
+		description = "Toggles notifications when a recurrent damage power-up appears",
+		position = 3
+	)
+	default boolean recurrentDamageNotification()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "ultimateforcenotification",
+		name = "Ultimate force notification",
+		description = "Toggles notifications when a ultimate force power-up appears",
+		position = 4
+	)
+	default boolean ultimateForceNotification()
```

Comment:
I don't think anyone will ever want notification for this.

Reply:
I’m with you on recurrent damage and especially ultimate force, but power surge is useful with eg SGS to refill your prayer. If I took the last two out, would that be good?

Reply:
Yeah, power surge is very nice, I think i pickup only that one actually. Maybe recurrent damage is good for points in some cases, but ultimate force should probably be removed.

Reply:
pts/hr wise recurrent damage and zapper are must-haves.

Reply:
I’ll take out ultimate force when I get home. Recurrent force is marginal, but maybe warrants having the option? The use case is AFKing, and I’m not sure it ups your overall dps enough to make it potentially worth it to bother running over to get it and stop attacking while you do, versus the time and attention you spend doing so.